features/fixtures/maze_runner/Assets/Scripts/Scenarios/Csharp/Events/SessionAfterStart.cs

```diff
@@ -8,7 +8,7 @@ public override void Run()
     {
         if (Application.platform.Equals(RuntimePlatform.OSXPlayer))
         {
-            Invoke("DoException",1);
+            Invoke("DoException",10);
```

Comment:
I notice this scenario has failed twice on CI for this PR (and on the first run it seemingly took down every scenario after it too).  

Was it flaky already, or could this have made things worse?  (I really can't see how it would affect subsequent scenarios though).

Reply:
The problem with this scenario, is that for MacOS to send a session, the fixture needs to start in the foreground, this cannot be reliably achieved on a remote VM. I have skipped the scenario on macos, this is not a change in coverage from the legacy tests.